.github/workflows/build_and_deploy.yml

```
@@ -0,0 +1,30 @@
+  push:
+    branches:
+      - main
+      - github-actions
+    tags:
+      - "*"
+
+jobs:
+  build_app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.9)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)